DeepSpeech.ipynb

```diff
@@ -172,7 +172,7 @@
     "export_version = 1\n",
     "remove_export = bool(int(os.environ.get('ds_remove_export', 0)))\n",
     "use_warpctc = bool(len(os.environ.get('ds_use_warpctc', '')))\n",
-    "default_stddev = float(os.environ.get('ds_default_stddev', 0.1))\n",
+    "default_stddev = float(os.environ.get('ds_default_stddev', 0.046875))\n",
```

Comment:
@reuben Yes the first and last should.

However, we didn't have this tweaked on a layer by layer basis before, so I thought tweaking layer by layer in this commit would be at orthogonal to the main concern of this commit, increasing the layer size.

I'll introduce issue #319 for it